src/frequenz/channels/_merge.py

```diff
@@ -13,13 +13,23 @@
 _T = TypeVar("_T")
 
 
-class Merge(Receiver[_T]):
-    """Merge messages coming from multiple channels into a single stream.
+def merge(*receivers: Receiver[_T]) -> Receiver[_T]:  # noqa: DOC502
+    """Merge messages coming from multiple receivers into a single stream.
+
+    Args:
+        *receivers: The receivers to merge.
+
+    Returns:
+        A receiver that merges the messages coming from multiple receivers into a
+            single stream.
+
+    Raises:
+        ValueError: if no receivers are provided.
```

Comment:
I'd be more precise here as it was done for `_Merge.__init__()`, otherwise it sounds to me like one receiver might be enough

```suggestion
        ValueError: if less than two receivers are provided.
```

Reply:
Ah, yeah, I forgot to update that docstring too, at first I checked that it wasn't empty, but then though it doesn't make much sense for only one receiver either.